rta/common.py

```diff
@@ -198,14 +195,23 @@ def close():
     return f, close
 
 
-def execute(command, hide_log=False, mute=False, timeout=30, wait=True, kill=False, drop=False, stdin=None,
-            shell=False, **kwargs):
+def execute(
+    command: Iterable,
+    hide_log=False,
+    mute=False,
+    timeout: int = 30,
+    wait=True,
+    kill=False,
+    drop=False,
+    stdin: Optional[Union[bytes, str]] = None,
+    shell=False,
+    **kwargs,
+):
```

Comment:
I wish black didn't do this but it's all good.

Reply:
agree - I am torn on some of the formatting, but should be fine for this as a cleanup 👍